container/CrabV2/CrabOTC/CrabOTC.tsx

```diff
@@ -345,24 +346,24 @@ const Withdraw: React.FC = () => {
                 </TableRow>
               </TableHead>
               <TableBody>
-                {Object.keys(userOtc?.data.bids).map((bidId, i) => (
-                  <TableRow key={bidId}>
+                {Object.keys(userOtc?.data.sortedBids).map((bidId) => (
+                  <TableRow key={Number(bidId)}>
                     <TableCell component="th" scope="row">
-                      {i + 1}
+                      {Number(bidId) + 1}
                     </TableCell>
                     <TableCell component="th" scope="row">
-                      {formatBigNumber(userOtc?.data.bids[bidId].order.quantity)}
+                      {formatBigNumber(userOtc?.data.sortedBids[Number(bidId)].order.quantity)}
                     </TableCell>
                     <TableCell component="th" scope="row">
-                      {formatBigNumber(userOtc?.data.bids[bidId].order.price, 18, 6)}
+                      {formatBigNumber(userOtc?.data.sortedBids[Number(bidId)].order.price, 18, 6)}
                     </TableCell>
                     <TableCell component="th" scope="row">
-                      {Date.now() < userOtc?.data.bids[bidId].order.expiry && Date.now() < userOtc?.data.expiry ? (
+                      {Date.now() < userOtc?.data.sortedBids[Number(bidId)].order.expiry && Date.now() < userOtc?.data.expiry ? (
                         <BoxLoadingButton
                           sx={{ width: 100 }}
                           size="small"
-                          onClick={() => withdrawCrab(userOtc?.data.bids[bidId], userOtc, i + 1)}
-                          loading={withdrawLoading === i + 1}
+                          onClick={() => withdrawCrab(userOtc?.data.sortedBids[Number(bidId)], userOtc, Number(bidId) + 1)}
```

Comment:
@DemolaJames have you tested that this is working in the inteneded way. Is it picking the right bid amount?
Can you test this on ropsten and post a loom on this PR?

Reply:
Yes i tested and it picks the right bid amount...will do a loom